com.playeveryware.eos/Runtime/EOS_SDK/Generated/ContinuanceToken.cs

```diff
@@ -50,9 +50,10 @@ public Result ToString(out Utf8String outBuffer)
 			// Call the EOS_ContinuanceToken_ToString function will a null buffer address to determine the correct buffer length.
 			// Discard the result of this call because it is not relevant. 
 			_ = Bindings.EOS_ContinuanceToken_ToString(InnerHandle, System.IntPtr.Zero, ref inOutBufferLength);
-			// PEW: End modify
-			
-			var funcResult = Bindings.EOS_ContinuanceToken_ToString(InnerHandle, outBufferAddress, ref inOutBufferLength);
+            outBufferAddress = Helper.AddAllocation(inOutBufferLength);
```

Comment:
Since this line is new - can you clarify why it was added? It seems identical to line 39, and it's not clear to me whether this was something that _should have_ been in the previous version or not.

Reply:
It should have been in the previous version. But more specifically, line 39 should have been moved down to where this line is at now.

Reply:
My thought is that since it is our edit to the SDK code, I would like to keep them in a block, and don't touch EOSSDK code if possible, it will be slightly easier to look at whenever an update happens

Reply:
This is true - but there is an earlier `Helper.AddAllocation` call, on line 39 which needs to be removed if this one is being kept.